treeherder/etl/buildbot.py

```diff
@@ -447,6 +448,7 @@
     {"regex": re.compile(r'b2g.*_eng.*_nightly'), "desc": "Unknown B2G Device Image Nightly (Engineering)"},
     {"regex": re.compile(r'b2g.*_emulator.*_nightly'), "desc": "B2G Emulator Image Nightly"},
     {"regex": re.compile(r'b2g.*_dolphin.*_nightly'), "desc": "Dolphin Device Image Nightly"},
+    {"regex": re.compile(r'b2g.*_dolphin-512.*_nightly'), "desc": "Dolphin-512 Device Image Nightly"},
```

Comment:
The regex in this file are order dependant - so you need to put the most specific ones first, or there will be false positives. ie: the dolphin-512 needs to go before dolphin. Whilst only some of the additions here need to go first, could you reorder them all, for consistency :-)

This change will fix the failing Travis run:
https://travis-ci.org/mozilla/treeherder-service/builds/48502114